### ISSUE TITLE
Common words reported as misspelled

### ISSUE THREAD

**Issue by kellyiscute** [iamcco/coc-spell-checker#58]
**Describe the bug**
Common words(language, mouse, import, etc.) are reported as misspelled on Arch Linux. Works fine on my mac.

**Expected behavior**

**Screenshots**
![image](https://github.com/iamcco/coc-spell-checker/assets/19303341/daa7a6b0-6d55-4534-8597-f15e1b4593f3)

**Desktop (please complete the following information):**
 - OS: Arch Linux

Log:

```
```
